db/migrate/20151103030415_add_username_password_to_user.rb

```diff
@@ -0,0 +1,6 @@
+class AddUsernamePasswordToUser < ActiveRecord::Migration
+  def change
+    add_column :users, :username, :string
```

Comment:
since you haven't deployed yet, you can add these to: `db/migrate/20151102162741_create_users.rb`